src/compas_fab/backends/pybullet/conftest.py

```diff
@@ -0,0 +1,5 @@
+import compas_fab.backends
```

Comment:
I am wondering if I should pull this conftest up to the top `compas_fab` directory rather than repeating it twice.  I'm conflicted because it's generally a terrible idea to alter behavior because one is running tests rather than in production and I don't want to encourage the use of this thing, but on the other hand, duplicate code is hard to maintain. Or maybe there is a different solution entirely?

Reply:
In the case of conftest files, I think it's fair to duplicate.